### PR TITLE
Add tables "place" and "district".

### PR DESCRIPTION
**Pull request by H-R-G-P**

### CODE REVIEW

**Comment by H-R-G-P** [comment]
Which one is true?
    (204,'в','Ляскавічы', 52, 2, 28.53528, 52.65861,NULL),
    (205,'в','Ляскавічы', 16, 2, 28.15417, 52.1175,NULL),

File: src/Entity/Place.php [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+    /**
+     * @ORM\ManyToOne(targetEntity=District::class, inversedBy="places")
+     */
+    private ?District $district_id;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private int $department_id;
```
Comment (b1514) [inline src/Entity/Place.php:38]: It's the link to entity

**Comment by H-R-G-P** [comment]
Nie zauważyłem commenty.

**Comment by H-R-G-P** [comment]
> Which one is true?
> (204,'в','Ляскавічы', 52, 2, 28.53528, 52.65861,NULL),
> (205,'в','Ляскавічы', 16, 2, 28.15417, 52.1175,NULL),
> and
> (234,'в','Будкі',88,2,29.26056,51.72111,NULL),
> (238,'в','Будкі',53,2,28.88861,51.78083,NULL),

I will add all places even if the names are the same.

**Pull request merged**